zwave_js_server/client.py

```diff
@@ -39,6 +40,11 @@
 from .model.log_message import LogMessage
 from .model.version import VersionInfo, VersionInfoDataType
 
+try:
+    import orjson as json
```

Comment:
Let's add this as an `extra` dependency that library users can select when installing the library.